examples/hybrid/sphere/baroclinic_wave_utilities.jl

```diff
@@ -408,7 +413,10 @@ function held_suarez_tendency!(Yₜ, Y, p, t)
     Δθ_z = FT(10)
     T_min = FT(200)
 
-    @. ᶜσ = ᶜp / MSLP
+    p_int_size = size(parent(ᶜp))
+    p_sfc = reshape(parent(p_sfc), (1, p_int_size[2:end]...))
+    parent(ᶜσ) .= parent(ᶜp) ./ parent(p_sfc)
```

Comment:
do we need parent here?

Reply:
Yeah I ran into the inconsistent `space instance` error otherwise